pkg/pipeline/extract/aggregate/aggregates.go

```diff
@@ -93,12 +97,16 @@ func (aggregates *Aggregates) cleanupExpiredEntries() {
 	}
 }
 
-func NewAggregatesFromConfig(definitions []api.AggregateDefinition) (Aggregates, error) {
+func NewAggregatesFromConfig(aggConfig *api.Aggregates) (Aggregates, error) {
 	aggregates := Aggregates{
-		expiryTime: defaultExpiryTime,
+		cleanupLoopTime:   cleanupLoopTime,
```

Comment:
Do we want to make `cleanupLoopTime` configurable as well or are we OK with keeping it 2 minutes?

Reply:
I think its OK.